### PR TITLE
"protocol_fee" e2e tests updated

### PR DESCRIPTION
**Pull request by DevRozaDev**

### CODE REVIEW

File: test/protocol_fee.test.ts [hunk 1]
```diff
@@ -1,371 +1,155 @@
...
-      expect(newFeeReceiverTokenXBalanceAfter).toBe(newFeeReceiverTokenXBalanceBefore + 1n)
-      expect(newFeeReceiverTokenYBalanceAfter).toBe(newFeeReceiverTokenYBalanceBefore)
-      expect(invariantTokenXBalanceAfter).toBe(invariantTokenXBalanceBefore - 1n)
-      expect(invariantTokenYBalanceAfter).toBe(invariantTokenYBalanceBefore)
-    }
+    expectError(
+      InvariantError.NotFeeReceiver,
+      invariant,
+      withdrawProtocolFee(invariant, notAdmin, tokenX, tokenY, fee, tickSpacing)
+    )
```
Comment (Sniezka1927) [inline test/protocol_fee.test.ts:153]: There is missing case when fee receiver is changed and new receiver tries to withdraw protcol fee

File: src/snippets.ts [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+import { balanceOf, deployInvariant } from './utils'
+import { PrivateKeyWallet } from '@alephium/web3-wallet'
+
+type TokenInstance = TokenFaucetInstance
+
+// 6% fee
+const fee = 6n * 10n ** (PercentageScale - 3n)
+const tickSpacing = 10n
+
+export const initBasicFeeTickSpacing = (): [bigint, bigint] => {
```
Comment (Sniezka1927) [inline src/snippets.ts:21]: You dont initialize anything, it would be more convinient to name it `getBasicFeeTickSpacing`

File: src/snippets.ts [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  await withdrawTokens(swapper, [tokenX, swapAmount])
+
+  const swapperTokenXBalanceBefore = await balanceOf(tokenX.contractId, swapper.address)
+  expect(swapperTokenXBalanceBefore).toBe(swapAmount)
+
+  const invariantBeforeExpectedBalance = {
+    tokenX: await balanceOf(tokenX.contractId, invariant.address),
+    tokenY: await balanceOf(tokenY.contractId, invariant.address)
+  }
+  expect(invariantBeforeExpectedBalance).toMatchObject({ tokenX: 500n, tokenY: 1000n })
```
Comment (Sniezka1927) [inline src/snippets.ts:101]: Why with Expected sufix?

File: src/snippets.ts [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+    tokenY,
+    fee,
+    tickSpacing,
+    true,
+    swapAmount,
+    true,
+    MinSqrtPrice
+  )
+
+  const swapperAfterExpectedBalance = {
```
Comment (Sniezka1927) [inline src/snippets.ts:116]: If this is expeced balance it should be hard coded, not queried from node

**Review by Sniezka1927 — CHANGES_REQUESTED** [review]

File: src/snippets.ts (hunk 3 above)
Comment (DevRozaDev) [reply src/snippets.ts:101]: good catch!

**Review by Sniezka1927 — APPROVED** [review]

**Review by wojciech-cichocki — APPROVED** [review]

**Pull request merged**